task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -2538,12 +2538,12 @@
 }
 
 @media (max-width: 1200px) {
-  .theme--light.v-navigation-drawer {
+  #TasksToolbar .theme--light.v-navigation-drawer {
```

Comment:
this rule is used for TasksToolbar or TasksDrawer ? cc @SaraBoutej

Reply:
Yes, I have taken care of that.

Reply:
it's used for both i think